content/en/docs/release_notes/wicklow.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Wicklow release notes
+linkTitle: Wicklow
+description: 30 September 2022

```

Comment:
Release date needs updating here and on JIRA. The other releases that were targeted for this week need to be pushed back to 2 weeks time.

Reply:
Done